### PR TITLE
BH eu2018: Unknown packer

### PR DESCRIPTION
**Pull request by enovella**
Closes issue https://github.com/rednaga/APKiD/issues/71

### CODE REVIEW

File: apkid/rules/dex/packers.yara [hunk 1]
```diff
@@ -243,3 +243,79 @@ rule jar_pack01 : packer
     is_dex and
     ($pre_jar and $jar_data and $post_jar)
 }
+
+rule unk_packer_a : packer
+{
+
+  meta:
+    description = "Unknown packer"
```
Comment (CalebFenton) [inline apkid/rules/dex/packers.yara]: The rule is excellent; it only needs a good name. It doesn't really matter, as long as it's unique. Something like:`GaoXor` (part of the obfuscated method name and Xor)

Comment (enovella) [reply apkid/rules/dex/packers.yara]: As you wish man

**Comment by enovella** [comment]
Done!

File: apkid/rules/dex/packers.yara (hunk 1 above)
Comment (CalebFenton) [reply apkid/rules/dex/packers.yara]: If we don't name them, we'll have Unknown packer 1, Unknown packer 2, etc. for days

File: apkid/rules/dex/packers.yara [hunk 2]
```diff
@@ -243,3 +243,79 @@ rule jar_pack01 : packer
     is_dex and
     ($pre_jar and $jar_data and $post_jar)
 }
+
+rule unk_packer_a : packer
```
Comment (CalebFenton) [inline apkid/rules/dex/packers.yara]: Could you change this also?

Comment (enovella) [reply apkid/rules/dex/packers.yara]: Sure

Comment (enovella) [reply apkid/rules/dex/packers.yara]: Done

**Pull request merged**